bin/lambda/tag.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# Using the env file is not mandatory. Variables can be already set in the current environment
+if [[ -f "env.d/lambda" ]]; then
+  source "env.d/lambda"
+fi
+
+function usage(){
+    echo -e "Usage: TAG ORIGIN_TAG
+Tag the existing image marsha/lambda:production to be pushed on ECR.
+To build the image marsha/lambda:production you can run ./bin/lambda/build.sh
+OPTIONS:
+  -h, --help       print this message
+"
+}
+
+# Tag the existing marsha/lambda:production to be pushed on ECR.
+function tag() {
+  tag=$1
+  origin_tag=$2
+  docker tag ${LAMBDA_IMAGE_NAME}:${origin_tag} ${LAMBDA_REPOSITORY_URL}/${LAMBDA_IMAGE_NAME}:${tag}
+}
+
+
+
+for i in "$@"
+do
+  case $i in
+    -h|--help|help)
+      usage "${0}"
+      exit 0
+      ;;
+  esac
+done
+
+tag ${1:-production} ${2:-production}
```

Comment:
Instead of having 3 scripts `build`, `tag` and `publish`, why not having a single `bin/lambda` script with 3 commands? This would avoid repetition.